### DISCUSSION TITLE
Argcd Login --Core Not working

### DISCUSSION THREAD

**Discussion by sabinayakc** [argoproj/argo-cd#12302] [Q&A] [answered]
I am trying to login using `argocd login --core` but having issues.

```
wsl-2:~$ argocd login --core
Context 'kubernetes' updated
wsl-2:~$ argocd repo list
FATA[0000] configmap "argocd-cm" not found
wsl-2:~$ argocd repo list --core
FATA[0000] configmap "argocd-cm" not found
wsl-2:~$
```
My ArgoCD version
```
wsl-2:~$ argocd version
argocd: v2.5.10+d311fad.dirty
  BuildDate: 2023-02-02T18:41:06Z
  GitCommit: d311fad538178ef97528edf7f09d1d5b57268f45
  GitTreeState: dirty
  GoVersion: go1.19.5
  Compiler: gc
  Platform: linux/amd64
FATA[0000] configmap "argocd-cm" not found
```

Is there something else I need to do or is this a bug?

**Comment by drpaneas** [comment]
What happens if you don't use 'core'? Are you able to login without it?

**Comment by sabinayakc** [comment]
I have a restriction where I can't use the server url and even if I did, its not accessible due to network policies. However, I do have access to the Kubernetes server via Kubectl. So i was trying to login to argocd cluster using the `--core` and create manifest.

I can verify that I have full cluster admin rights since I can create and delete manifest using simple kubectl commands.

**Comment by sabinayakc** [comment, marked as answer]
I had to set the context to align with the `argocd` namespace.

`kubectl config set-context --current --namespace=argocd`

**Reply by naveenps163** [reply]
please check argocd namespace is available or  not, kubectl get namespace, if name space is available then u can type above command
